pages/services/edge-lb/1.2/upgrading/index.md

```diff
@@ -27,12 +27,12 @@ Perform an Edge-LB upgrade by following this procedure.
 
     ```bash
     dcos package repo add --index=0 edgelb-aws \
-      https://<AWS S3 bucket>/stub-universe-edgelb.json
+      https://<insert download link>/stub-universe-edgelb.json
     dcos package repo add --index=0 edgelb-pool-aws \
-      https://<AWS S3 bucket>/stub-universe-edgelb-pool.json
+      https://<insert download link>/stub-universe-edgelb-pool.json
     ```
 
-1. Install the new `apiserver`. Use the service account created when installing the previous version; see [Edge-LB Installation Guide](/services/edge-lb/1.1/installing) for more information.
+1. Install the new `apiserver`. Use the service account created when installing the previous version; see [Edge-LB Installation Guide](/services/edge-lb/1.1/installing) for more information. The configuration file below matches the one created while following the install instructions.
```

Comment:
Which instal instructions? "these install instructions", "the install instructions above", "the install instructions from the Installation Guide"